<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XyzWidget</class>
 <widget class="QWidget" name="XyzWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>352</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>XyzWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QCustomPlot" name="customPlot2" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot1" native="true"/>
    </item>
    <item row="1" column="0">
     <widget class="QCustomPlot" name="customPlot3" native="true"/>
    </item>
    <item row="1" column="1">
     <widget class="QCustomPlot" name="customPlot4" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
